server/sonar-webserver-core/src/main/java/org/sonar/server/telemetry/TelemetryDaemon.java

```diff
@@ -114,7 +114,7 @@ private Runnable telemetryCommand() {
           updateTelemetryProps(now);
         }
       } catch (Exception e) {
-        LOG.debug("Error while checking SonarQube statistics: {}", e);
+        LOG.debug("Error while checking SonarQube statistics: {}", e.getMessage());
```

Comment:
```suggestion
        LOG.debug("Error while checking SonarQube statistics: {}", e.getMessage(), e);
```

Reply:
I did it, 🤝.
Thank you for the feedback @pierre-guillot-gh!